specification/components/schemas/extensions/Extension-eRS-ServiceSummaryView.yaml

```diff
@@ -461,29 +461,6 @@ properties:
             valueBoolean:
               type: boolean
               example: true
-            valueReference:
```

Comment:
It looks like this extension is used as part of the response from patient service search so I think this organisation reference will still be returned.

Reply:
Ah I remember now, this part of the OAS file is currently incorrect on develop as the `ReferralLetterRequired` extension returned by the API will only be a boolean, not a reference.